blueprint/index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Develop a React App with Typescript that uses the Genesys Cloud Platform SDK
+author: Jacob Shaw
```

Comment:
This needs to match your author data in the dev center (to be added). Should be `jacob.shaw`.